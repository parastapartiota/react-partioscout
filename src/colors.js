// MAIN
export const BLACK = '#000000';
export const WHITE = '#ffffff';
export const BLUE = '#253764';

export const Main = {
  BLACK,
  WHITE,
  BLUE
};

// EXTRA
export const ORANGE = '#efa01e';
export const YELLOW = '#f0e005';
export const GREEN = '#14a54b';
export const LIGHT_BLUE = '#28a9e1';
export const RED = '#28a9e1';

export const Extra = {
  ORANGE,
  YELLOW,
  GREEN,
  LIGHT_BLUE,
  RED
};
