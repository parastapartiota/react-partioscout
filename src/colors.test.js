import * as Colors from './colors';
describe('Colors', () => {
  test('BLACK', () => {
    expect(Colors.BLACK).toMatchSnapshot();
  });

  test('WHITE', () => {
    expect(Colors.WHITE).toMatchSnapshot();
  });

  test('BLUE', () => {
    expect(Colors.BLUE).toMatchSnapshot();
  });

  test('YELLOW', () => {
    expect(Colors.YELLOW).toMatchSnapshot();
  });

  test('ORANGE', () => {
    expect(Colors.ORANGE).toMatchSnapshot();
  });

  test('GREEN', () => {
    expect(Colors.GREEN).toMatchSnapshot();
  });

  test('LIGHT_BLUE', () => {
    expect(Colors.LIGHT_BLUE).toMatchSnapshot();
  });

  test('RED', () => {
    expect(Colors.RED).toMatchSnapshot();
  });
});
