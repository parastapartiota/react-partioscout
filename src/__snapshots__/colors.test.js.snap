// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors BLACK 1`] = `"#000000"`;

exports[`Colors BLUE 1`] = `"#253764"`;

exports[`Colors GREEN 1`] = `"#14a54b"`;

exports[`Colors LIGHT_BLUE 1`] = `"#28a9e1"`;

exports[`Colors ORANGE 1`] = `"#efa01e"`;

exports[`Colors RED 1`] = `"#28a9e1"`;

exports[`Colors WHITE 1`] = `"#ffffff"`;

exports[`Colors YELLOW 1`] = `"#f0e005"`;
